### ISSUE TITLE
Import Icon

### ISSUE THREAD

**Issue by polarkids** [rekit/antd-form-builder#28]
HI, i just switch to Antd@4.0.3 and i got this error 

```
./node_modules/antd-form-builder/dist/index.es.js
Attempted import error: 'Icon' is not exported from 'antd'.
```

And i'm using antd-form-builder@1.1.3

How can i fix it ? 

Thanks

**Comment by supnate** [comment]
Please wait for the new version for antd v4 #17

**Comment by supnate** [comment]
Fixed in 2.0.
